framework/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ Yii Framework 2 Change Log
 - Enh #19841: Allow jQuery 3.7 to be installed (wouter90)
 - Enh #19853: Added support for default value for `\yii\helpers\Console::select()` (rhertogh)
 - Bug #19868: Added whitespace sanitation for tests, due to updates in ICU 72 (schmunk42)
-
```

Comment:
New line is missing (there should be 2 empty lines after the last text)

Reply:
Added a correction.